lib/cmap/auth/gssapi.js

```diff
@@ -45,14 +44,6 @@ class GSSAPI extends AuthProvider {
    * @method
    */
   auth(sendAuthCommand, connections, credentials, callback) {
-    if (kerberos == null) {
```

Comment:
I think this is a behavioral change. Initially, if kerberos was opted for and wasn't present it would bail early here and callback with the error. Now, it seems we will bail somewhere later in the process, most likely synchronously in the `GSSAPIInitialize` call above. I think we'll need a test around this, and my guess is that you'll need to make the "error module" identifiable outside of all calls to the proxy throwing an error.